lua/keymap/completion.lua

```diff
@@ -50,6 +51,10 @@ function mapping.lsp(buf)
 		["n|gd"] = map_cr("Glance definitions"):with_silent():with_buffer(buf):with_desc("lsp: Preview definition"),
 		["n|gD"] = map_cr("Lspsaga goto_definition"):with_silent():with_buffer(buf):with_desc("lsp: Goto definition"),
 		["n|gh"] = map_cr("Glance references"):with_silent():with_buffer(buf):with_desc("lsp: Show reference"),
+		["n|gi"] = map_cu("lua vim.lsp.buf.implementation()")
```

Comment:
```suggestion
		["n|gm"] = map_cr("Glance implementations")
```
(Glace provides this [feature](https://github.com/DNLHC/glance.nvim?tab=readme-ov-file#commands))

qq: Is using `gi` really a good choice here? Because at least in my workflow, I sometimes need it:
```console
							*gi*
gi			Insert text in the same position as where Insert mode
			was stopped last time in the current buffer.
			This uses the |'^| mark.  It's different from "`^i"
			when the mark is past the end of the line.
			The position is corrected for inserted/deleted lines,
			but NOT for inserted/deleted characters.
			When the |:keepjumps| command modifier is used the |'^|
			mark won't be changed.
```